animeal/AppDelegate.swift

```diff
@@ -29,6 +30,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AppDelegateProtocol {
             _ = $0.registerApplication(application, didFinishLaunchingWithOptions: launchOptions)
         }
         logInfo("[APP] \(#function)")
+
+        // For Debug purposes
+        // AWSDDLog.sharedInstance.logLevel = .verbose
+        // AWSDDLog.add(AWSDDTTYLogger.sharedInstance)
```

Comment:
nit: Do we need this?

Reply:
I believe yes, this pies of code add's a lot of helpful information in output. Helps me while debugging, could be helpful for some one else.